README.md

```diff
@@ -1,5 +1,12 @@
 # Digital-Twins
 
+## Setup
+Setup a conda environment to run in using the following command run from the repository folder:
```

Comment:
great, thanks for documenting how to set up the environment.

Reply:
Thanks :)